pkg/scheduler/scheduler.go

```diff
@@ -309,6 +309,10 @@ func New(client clientset.Interface,
 		return nil, errors.New("at least one profile is required")
 	}
 
+	enqueuePluginMap := make(map[string][]framework.PreEnqueuePlugin)
```

Comment:
Should we move the PreEnqueue extension point itself behind the feature gate? How can users disable the PreEnqueue extension point if the PreEnqueue extension point itself (not `.spec.schedulingGate` or the scheduling gate plugin) has the bug.
(Please ignore if this topic is already discussed. I haven't followed all discussions 🙏 )

Reply:
See https://github.com/kubernetes/kubernetes/pull/113275#discussion_r1014341424.